swagger/swagger-generator/generator-core/src/test/java/io/servicecomb/swagger/extend/property/creator/TestPartPropertyCreator.java

```diff
@@ -0,0 +1,41 @@
+/*
```

Comment:
Please update the license header with the new one.

Reply:
done